IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -241,7 +275,21 @@ - (MSIDRefreshToken *)getRefreshableTokenWithAccount:(MSIDAccountIdentifier *)ac
 
         if (refreshToken)
         {
-            MSID_LOG_WITH_CTX(MSIDLogLevelVerbose, context, @"(Default accessor) Found %@refresh token by home account id", credentialType == MSIDPrimaryRefreshTokenType ? @"primary " : @"");
+            NSString *credentialTypeString = nil;
+            if (credentialType == MSIDPrimaryRefreshTokenType)
```

Comment:
can we read this directly from refreshToken? i.e:

`[refreshToken credentialTypeString]`

We will be able to reuse it in another places too.

Reply:
Updated with existing method `[MSIDCredentialTypeHelpers credentialTypeAsString:credentialType]`